src/utils.c

```diff
@@ -931,3 +931,69 @@ ts_relation_size(Oid relid)
 
 	return ret;
 }
+
+#define STR_VALUE(str) #str
+#define NODE_CASE(name)                                                                            \
+	case T_##name:                                                                                 \
+		return STR_VALUE(name)
+
+/*
+ * Return a string with the name of the node.
+ *
+ */
+const char *
+ts_get_node_name(Path *path)
```

Comment:
suggest renaming as ts_get_path_node_name.

Reply:
changed the signature to take Node* instead